docs/development.md

```diff
@@ -133,6 +133,7 @@ It is recommended to use Firefox 46.0 as this is the browser used in CI build (T
 
 **NOTE**
 > The dev server sets its time zone to UTC at startup.
+> View and edit `test.selenium.browser` in `src/test/resources/test.properties` to change browser for testing (default is firefox)
```

Comment:
You should not place instructions here. This is a special note to let developers know the behavior of the dev server.

Reply:
Hi, I meant to say that "if you wish to change the browser used for testing, you may do so here." I think it is helpful because as a first timer I didn't see information in this article.

Reply:
I'm afraid you are not catching what I am trying to say. The note here is not part of the flow but is meant to bring special attention to the user.

Whereas the instructions for setting the browser is supposed to be part of the flow and should be placed alongside say
> TEAMMATES automated testing requires Firefox or Chrome (works on Windows and OS X). It is recommended to use Firefox 46.0 as this is the browser used in CI build (Travis/AppVeyor).

Reply:
Kindly see updated.